torch_audiomentations/augmentations/impulse_response.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import random
+import torch
+from torch_audiomentations.core.transforms_interface import BasicTransform
+from torch_audiomentations.utils.convolution import convolve
+from torch_audiomentations.utils.file import find_files, load_audio
```

Comment:
Shouldn't imports be relative?

Reply:
If it would be up to me, I would use absolute imports because I let my IDE (Jetbrains IntelliJ) take care of adding imports (via a keyboard shortcut), and it uses absolute imports.

Also, a quick google search returns this:

![bilde](https://user-images.githubusercontent.com/1470603/94139247-9e5ae680-fe69-11ea-8882-43762f4e66dd.png)

Reply:
I'll have a look at some other serious python libraries and see what they do

Reply:
Stripe: Absolute imports https://github.com/stripe/stripe-python/blob/master/stripe/api_resources/bank_account.py#L5

Django: Absolute imports 
https://github.com/django/django/blob/master/django/core/paginator.py#L6

Scikit-learn: Relative imports https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/impute/_iterative.py#L11

Keras: Relative imports
https://github.com/keras-team/keras/blob/fb7f49ef5b07f2ceee1d2d6c45f273df6672734c/keras/losses.py#L9

Pandas: Absolute imports https://github.com/pandas-dev/pandas/blob/master/pandas/core/aggregation.py#L23

Flask: Relative imports https://github.com/pallets/flask/blob/master/src/flask/app.py#L32

Looks like a 50/50 to me ^^

Reply:
Those are for the old relative imports, without the `.`. 
The new relative imports use the `.`, they are called explicit relative imports and the part of PEP8 that discourages relative imports has been removed 

Quote from Guido himself
> Isn't this mostly historical? Until the new relative-import syntax was
implemented there were various problems with relative imports. The
short-term solution was to recommend not using them. The long-term
solution was to implement an unambiguous syntax. Now it is time to
withdraw the anti-recommendation. Of course, without going overboard
-- I still find them an acquired taste; but they have their place.

For example, keras, pytorch and albumentations use explicit relative imports. 
pytorch-lightning uses absolute imports. 

I think most of the resources around this debate on Google are quite outdated.

Reply:
Then it's probably fine to keep like this. 
Somehow I mainly wrote code that used explicit relative imports at the beginning and thought this is the norm, happy to know that the other way is fine ! 
Let's stick with this then if you'd like !

Reply:
Good point. Well, either way works for me 😄